misk/src/main/kotlin/misk/web/MiskWebModule.kt

```diff
@@ -113,10 +114,14 @@ class MiskWebModule(private val config: WebConfig) : KAbstractModule() {
     multibind<NetworkInterceptor.Factory>(MiskDefault::class)
         .to<ExceptionHandlingInterceptor.Factory>()
 
-    // Optionally log request and response details
-    multibind<ApplicationInterceptor.Factory>(MiskDefault::class)
+    // Log request and response details
```

Comment:
isn't this optional as well based on sampling?

Reply:
I mistakenly thought this was opt-out, but it's opt-in. 👍